README.md

```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
 * `build.sh` runs stuff (which downloads uno if needed), and builds all packages.
 * `test.sh` runs all tests.
 
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
+
+```
+Packages.SourcePaths += <path-to-fuselibs>/Source
+```
+
+You (obviously) need to replace `<path-to-fuselibs>` above with the actual path to your fuselibs checkout.
```

Comment:
@mortoray: I believe fuse doesn't use Packages.**Source**Path, and I believe uno prefers using the source-path. This is reported to work for users...